web/onos-gui/src/app/onos-config/models-list/models-list.component.ts

```diff
@@ -107,4 +110,11 @@ export class ModelsListComponent extends TableBaseImpl implements OnInit, OnDest
         }
         this.newConfigTitle = '';
     }
+
+    // onSort(colname: string) {
```

Comment:
this is probably not needed

Reply:
these were there from before -  I will remove it

Reply:
I don't know - it showed up in green - thanks for removing